crates/oxc_cli/src/walk.rs

```diff
@@ -50,15 +50,17 @@ impl ignore::ParallelVisitor for WalkCollector {
 }
 
 impl Walk {
+    /// Will not canonicalize paths.
```

Comment:
"Paths are not canonicalize", we seldom you `we` in comments.

Reply:
I'm sorry, I don't understand what you meant by this

Reply:
Sorry I misread it.